ports/stm32/boards/OPENMV_N6/mpconfigboard.mk

```diff
@@ -12,15 +12,26 @@ STM32_N6_HEADER_VERSION = 2.3
 DKEL = $(CUBE_PROG_BASE)/bin/ExternalLoader/MX25UM51245G_STM32N6570-NUCLEO.stldr
 
 ifeq ($(USE_MBOOT),1)
-LD_FILES = boards/stm32n657x0.ld boards/common_bl.ld
+LD_FILES = boards/stm32n657x0.ld boards/OPENMV_N6/board.ld boards/common_bl.ld
 TEXT0_ADDR = 0x70080000
+ifneq ($(BUILDING_MBOOT),1)
+CFLAGS += -DMICROPY_HW_RUNS_FROM_EXT_FLASH=1
+endif
 else
-LD_FILES = boards/stm32n657x0.ld boards/common_basic.ld
+LD_FILES = boards/stm32n657x0.ld boards/OPENMV_N6/board.ld boards/common_basic.ld
 TEXT0_ADDR = 0x34180400
 endif
 
 # Bootloader settings
 MBOOT_TEXT0_ADDR = 0x34180400
-#MBOOT_TEXT0_ADDR = 0x70080000
 MBOOT_LD_FILES = ../boards/OPENMV_N6/mboot_memory.ld stm32_sections.ld
 
+# MicroPython settings
+MICROPY_PY_BLUETOOTH ?= 1
+MICROPY_BLUETOOTH_NIMBLE ?= 1
+MICROPY_BLUETOOTH_BTSTACK ?= 0
+MICROPY_PY_LWIP = 1
+MICROPY_PY_NETWORK_CYW43 = 1
+MICROPY_PY_SSL = 1
+MICROPY_SSL_MBEDTLS = 1
+MICROPY_VFS_LFS2 = 1
```

Comment:
I need to be able to disable mboot `USE_MBOOT ?= 1` and also we never use LFS2, just fat.

Reply:
But what if you don't want to build mboot, yet still want a flash-based image? Can `USE_MBOOT=1` define something like `MICROPY_FLASH_BASED=1`? For example:
```
USE_MBOOT ?= 1
MICROPY_FLASH_BASED ?= $(USE_MBOOT)
```

 This way I can define `USE_MBOOT=0 MICROPY_FLASH_BASED=1` to get a flash-based image without mboot.